routerchain/chain.json

```diff
@@ -0,0 +1,166 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "router chain",
+    "chain_type": "cosmos",
+    "chain_id": "9600",
+    "pretty_name": "Router Protocol",
+    "status": "live",
+    "network_type": "mainnet",
+    "website": "https://www.routerprotocol.com/",
+    "bech32_prefix": "router",
+    "daemon_name": "routerd",
+    "key_algos": [
+      "secp256k1"
+    ],
+    "slip44": 118,
+    "fees": {
+      "fee_tokens": [
+        {
+          "denom": "route",
+          "fixed_min_gas_price": 0.006,
+          "low_gas_price": 0.006,
+          "average_gas_price": 0.006,
+          "high_gas_price": 0.01
+        }
+      ]
+    },
+    "staking": {
+      "staking_tokens": [
+        {
+          "denom": "route"
+        }
+      ]
+    },
+    "codebase": {
+      "versions": [
+        {
+          "name": "v2.1.1",
+          "recommended_version": "v2.1.1",
+          "compatible_versions": [
+            "v2.1.1"
+          ]
+        }
+      ],
+      "binaries": {
```

Comment:
please copy from binaries to cosmwasm_enabled inside the versions array as well.